### ISSUE TITLE
[1.30] java.lang.NullPointerException: Tesselating block model

### ISSUE THREAD

**Issue by savosdarkran** [NillerMedDild/Enigmatica2#14]
#### Issue Description
When looking at specific block in world the game crashes.

#### What happens
When I go to mine with a tinker's hammer or tinker's pick I get a crash.  I went out of that area and broke dirt with an Osmium shovel.  Went back to the same area and broke the stone facing a different side of where I was before and it was fine.  So I then tried another place where it crashed and and on the same angle it seems to be okay.  Perhaps breaking the dirt reset something in the area?  This happened in both single player and multiplayer on the same world ( I ported my single player save to a server ).  Pretty much resolved now, but maybe you can see something in the crash report I cannot.

#### What you expected to happen
I can mine without issue.

#### Modpack version (Do not use "latest")
1.30

#### Your most recent crash report or log file where the issue was present
https://paste.ee/p/mLVbe

**Comment by NillerMedDild** [comment]
Weird that it fixed itself, but awesome. 
It seems to be a crash related to the lighting of a nearby ArchitectureCraft block.

**Comment by savosdarkran** [comment]
https://github.com/elytra/ArchitectureCraft/issues/28  This may be the thing.

**Comment by savosdarkran** [comment]
I logged in twice and both times I crashed.  I posted one crash report.  Did you want to see the other?

I disabled foamfix loaded the instance and was able to log in and stay in world for about 20 seconds.  I will do a more thorough test when I am back in front of my computer.  I did this remotely ( lagggg ).

I do have an AE2 grindstone and crank near my spawn in.

Thank you for the quick response.  Will let you know my results when I am home again.
